lib/steps/convert-document.js

```diff
@@ -0,0 +1,18 @@
+const convert = require('@vtfk/minelev-document-converter')
+const { logger } = require('@vtfk/logger')
+
+module.exports = async data => {
+  try {
+    const lang = 'nb'
+    logger('info', ['convert document', data._id, 'language', lang, 'start'])
+    // TODO: add language support...
```

Comment:
Hvilke felter er det som er avhengige av språket her? Tror det blir problematisk å få til at den henter inn språk fra KRR så tidlig i prosessen, ser heller for meg at det kan gjøres før/etter DSF-steget.

Dersom det som blir repacket som har flere språk primært er i bruk i dokumentmalene, tror jeg ikke det er noe problem om language er "hardkodet" til `nb` her fordi malene bruker `newDocument`-objektet, som innholder alle språkene, og når den først skal generere dokumenter vil vi jo (forhøpentligvis) ha språket tilgjengelig.

Reply:
Språk gis med for å hente riktig språk for følgende felter:
- samtaleCategories (gir pt bare tilbake i `nb` uansett...)
- period
- behaviourCategories
- gradesCategories
- coursesList
- orderCategories